templates/project/.github/workflows/qa.yaml.twig

```diff
@@ -0,0 +1,29 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+
+name: Quality assurance
+
+on:
+  push:
+    branches:
+{% for branch in project.branches %}
+      - {{ branch.name }}
+{% endfor %}
+  pull_request:
+
+jobs:
+  phpstan:
+    name: PHPStan
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: PHPStan
+        uses: docker://oskarstark/phpstan-ga
+        env:
+          REQUIRE_DEV: true
+        with:
+          args: analyse -c .phpstan/phpstan.neon.dist
```

Comment:
The file should not be in a subfolder

Reply:
I used the one from the initial PR from @core23 

@core23 shall we change that?

Reply:
I copy & pasted it from the admin bundle, which provides some custom stubs: https://github.com/sonata-project/SonataAdminBundle/tree/3.x/.phpstan

Reply:
Not all the bundles / branches uses same structure

Reply:
And we just merge this: https://github.com/sonata-project/dev-kit/blob/49ca452d26bb3b93d6a41b5e95b1a6419214dc79/templates/project/.gitattributes.twig#L11

Reply:
Maybe we can find a global structure where we store all our tool configs? E.g. `/tools/$TOOL/.config`

Reply:
If this is our current structure, lets keep it, lets run dev-kit, and improve it if needed

Reply:
It's not the current structure for SonataAdmin master branch.
And it's not the same structure for psalm.

IMHO we should have consistency and move the phpstan.neon.dist to the root folder.

Reply:
Not only for admin, the mongodb bundle does have the file on the root folder